README.md

```diff
@@ -5,6 +5,7 @@ Guides is a public repo for storing, among other things, public configuration fi
 ### Style guide configurations
 
 [Ruby](/style/ruby/.rubocop.yml)
+[JavaScript](https://github.com/BetterOfficeApps/eslint-config-lendesk)
```

Comment:
We should be storing these in a .eslintrc file

I know you are following up with @RafeHatfield about this 👍

Reply:
Just adding this here for future selves:

Rafe has approved moving all linter configs into a public repo.

Carry on.